runtime/auth/aws-signing-common/common/src/aws/smithy/kotlin/runtime/auth/awssigning/AwsChunkedSource.kt

```diff
@@ -13,15 +12,13 @@ import aws.smithy.kotlin.runtime.io.SdkSource
 import aws.smithy.kotlin.runtime.io.buffer
 import kotlinx.coroutines.runBlocking
 
-// TODO - can be shared with Kotlin/Native but not JS
```

Comment:
Note: this was moved to `expect`/`actual` some time ago but I moved it back to common because I was running into issues getting the `actual` native implementation to work. 

I figured we could move it back to `expect`/`actual` when we add JS support? Is that backwards-compatible?

Reply:
I think we will be able to move things around and still keep binary compat for JVM here.